### ISSUE TITLE
Error with Git bash when installing Python environment: UnicodeDecodeError: 'utf8' codec can't decode byte 0x82 in position 928: invalid start byte

### ISSUE THREAD

**Issue by Lucas-C** [pre-commit/pre-commit#330]
```
$ git commit -m "Initial commit"
[INFO] Initializing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Installing environment for git://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Initializing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Installing environment for git://github.com/Lucas-C/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
An unexpected error has occurred: UnicodeDecodeError: 'utf8' codec can't decode byte 0x82 in position 928: invalid start byte
Check the log at ~/.pre-commit/pre-commit.log

$ cat ~/.pre-commit/pre-commit.log
An unexpected error has occurred: UnicodeDecodeError: 'utf8' codec can't decode byte 0x82 in position 928: invalid start byte
Traceback (most recent call last):
  File "c:\devhome\tools\Python27\lib\site-packages\pre_commit\error_handler.py", line 36, in error_handler
    yield
  File "c:\devhome\tools\Python27\lib\site-packages\pre_commit\main.py", line 147, in main
    return run(runner, args)
  File "c:\devhome\tools\Python27\lib\site-packages\pre_commit\commands\run.py", line 195, in run
    repo_hooks = list(get_repo_hooks(runner))
  File "c:\devhome\tools\Python27\lib\site-packages\pre_commit\commands\run.py", line 141, in get_repo_hooks
    for repo in runner.repositories:
  File "c:\devhome\tools\Python27\lib\site-packages\cached_property.py", line 26, in __get__
    value = obj.__dict__[self.func.__name__] = self.func(obj)
  File "c:\devhome\tools\Python27\lib\site-packages\pre_commit\runner.py", line 47, in repositories
    repository.require_installed()
  File "c:\devhome\tools\Python27\lib\site-packages\pre_commit\repository.py", line 115, in require_installed
    self.install()
  File "c:\devhome\tools\Python27\lib\site-packages\pre_commit\repository.py", line 185, in install
    self.additional_dependencies[language_name][language_version],
  File "c:\devhome\tools\Python27\lib\site-packages\pre_commit\languages\python.py", line 66, in install_environment
    env.run("cd '{prefix}' && pip install .")
  File "c:\devhome\tools\Python27\lib\site-packages\pre_commit\languages\helpers.py", line 51, in run
    ['bash', '-c', ' '.join([self.env_prefix, cmd])], **kwargs
  File "c:\devhome\tools\Python27\lib\site-packages\pre_commit\prefixed_command_runner.py", line 39, in run
    return cmd_output(*replaced_cmd, __popen=self.__popen, **kwargs)
  File "c:\devhome\tools\Python27\lib\site-packages\pre_commit\util.py", line 175, in cmd_output
    stdout = stdout.decode(encoding)
  File "c:\devhome\tools\Python27\lib\encodings\utf_8.py", line 16, in decode
    return codecs.utf_8_decode(input, errors, True)
UnicodeDecodeError: 'utf8' codec can't decode byte 0x82 in position 928: invalid start byte
```

I reproduced the internal system calls mad by `pre-commit` here:
```
$ c:\\devhome\\tools\\Python27\\python.exe -m virtualenv d:/Users/lucas_cimon\\.pre-commit\\repozp6fda\\py_env-default
New python executable in d:/Users/lucas_cimon\.pre-commit\repozp6fda\py_env-default\Scripts\python.exe
Installing setuptools, pip, wheel...done.

$ bash -c ". 'd:/Users/lucas_cimon\\.pre-commit\\repozp6fda\\py_env-default\\Scripts\\activate' && cd 'd:/Users/lucas_cimon\\.pre-commit\\repozp6fda\\' && pip install ."
Processing d:\users\lucas_cimon\.pre-commit\repozp6fda
Collecting lxml (from pre-commit-hooks==0.0.1)
  Using cached lxml-3.5.0.tar.gz
Collecting tinycss2 (from pre-commit-hooks==0.0.1)
Collecting webencodings>=0.4 (from tinycss2->pre-commit-hooks==0.0.1)
Building wheels for collected packages: lxml
  Running setup.py bdist_wheel for lxml
  Complete output from command d:\users\lucas_cimon\.pre-commit\repozp6fda\py_env-default\scripts\python.exe -c "import setuptools;__file__='c:\\temp\\pip-build-vjp3v8\\lxml\\setup.py';exec(compile(open(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" bdist_wheel -d c:\temp\tmpnpxj1bpip-wheel-:
  Building lxml version 3.5.0.
  Building without Cython.
  ERROR: 'xslt-config' n'est pas reconnu en tant que commande interne
  ou externe, un programme exécutable ou un fichier de commandes.

  ** make sure the development packages of libxml2 and libxslt are installed **

  Using build configuration of libxslt
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build\lib.win-amd64-2.7
  creating build\lib.win-amd64-2.7\lxml
  copying src\lxml\builder.py -> build\lib.win-amd64-2.7\lxml
  copying src\lxml\cssselect.py -> build\lib.win-amd64-2.7\lxml
  copying src\lxml\doctestcompare.py -> build\lib.win-amd64-2.7\lxml
  copying src\lxml\ElementInclude.py -> build\lib.win-amd64-2.7\lxml
  copying src\lxml\pyclasslookup.py -> build\lib.win-amd64-2.7\lxml
  copying src\lxml\sax.py -> build\lib.win-amd64-2.7\lxml
  copying src\lxml\usedoctest.py -> build\lib.win-amd64-2.7\lxml
  copying src\lxml\_elementpath.py -> build\lib.win-amd64-2.7\lxml
  copying src\lxml\__init__.py -> build\lib.win-amd64-2.7\lxml
  creating build\lib.win-amd64-2.7\lxml\includes
  copying src\lxml\includes\__init__.py -> build\lib.win-amd64-2.7\lxml\includes

  creating build\lib.win-amd64-2.7\lxml\html
  copying src\lxml\html\builder.py -> build\lib.win-amd64-2.7\lxml\html
  copying src\lxml\html\clean.py -> build\lib.win-amd64-2.7\lxml\html
  copying src\lxml\html\defs.py -> build\lib.win-amd64-2.7\lxml\html
  copying src\lxml\html\diff.py -> build\lib.win-amd64-2.7\lxml\html
  copying src\lxml\html\ElementSoup.py -> build\lib.win-amd64-2.7\lxml\html
  copying src\lxml\html\formfill.py -> build\lib.win-amd64-2.7\lxml\html
  copying src\lxml\html\html5parser.py -> build\lib.win-amd64-2.7\lxml\html
  copying src\lxml\html\soupparser.py -> build\lib.win-amd64-2.7\lxml\html
  copying src\lxml\html\usedoctest.py -> build\lib.win-amd64-2.7\lxml\html
  copying src\lxml\html\_diffcommand.py -> build\lib.win-amd64-2.7\lxml\html
  copying src\lxml\html\_html5builder.py -> build\lib.win-amd64-2.7\lxml\html
  copying src\lxml\html\_setmixin.py -> build\lib.win-amd64-2.7\lxml\html
  copying src\lxml\html\__init__.py -> build\lib.win-amd64-2.7\lxml\html
  creating build\lib.win-amd64-2.7\lxml\isoschematron
  copying src\lxml\isoschematron\__init__.py -> build\lib.win-amd64-2.7\lxml\isoschematron
  copying src\lxml\lxml.etree.h -> build\lib.win-amd64-2.7\lxml
  copying src\lxml\lxml.etree_api.h -> build\lib.win-amd64-2.7\lxml
  copying src\lxml\includes\c14n.pxd -> build\lib.win-amd64-2.7\lxml\includes
  copying src\lxml\includes\config.pxd -> build\lib.win-amd64-2.7\lxml\includes
  copying src\lxml\includes\dtdvalid.pxd -> build\lib.win-amd64-2.7\lxml\includes
  copying src\lxml\includes\etreepublic.pxd -> build\lib.win-amd64-2.7\lxml\includes
  copying src\lxml\includes\htmlparser.pxd -> build\lib.win-amd64-2.7\lxml\includes
  copying src\lxml\includes\relaxng.pxd -> build\lib.win-amd64-2.7\lxml\includes

  copying src\lxml\includes\schematron.pxd -> build\lib.win-amd64-2.7\lxml\includes
  copying src\lxml\includes\tree.pxd -> build\lib.win-amd64-2.7\lxml\includes
  copying src\lxml\includes\uri.pxd -> build\lib.win-amd64-2.7\lxml\includes
  copying src\lxml\includes\xinclude.pxd -> build\lib.win-amd64-2.7\lxml\includes
  copying src\lxml\includes\xmlerror.pxd -> build\lib.win-amd64-2.7\lxml\includes
  copying src\lxml\includes\xmlparser.pxd -> build\lib.win-amd64-2.7\lxml\includes
  copying src\lxml\includes\xmlschema.pxd -> build\lib.win-amd64-2.7\lxml\includes
  copying src\lxml\includes\xpath.pxd -> build\lib.win-amd64-2.7\lxml\includes
  copying src\lxml\includes\xslt.pxd -> build\lib.win-amd64-2.7\lxml\includes
  copying src\lxml\includes\etree_defs.h -> build\lib.win-amd64-2.7\lxml\includes
  copying src\lxml\includes\lxml-version.h -> build\lib.win-amd64-2.7\lxml\includes
  creating build\lib.win-amd64-2.7\lxml\isoschematron\resources
  creating build\lib.win-amd64-2.7\lxml\isoschematron\resources\rng
  copying src\lxml\isoschematron\resources\rng\iso-schematron.rng -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\rng
  creating build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl
  copying src\lxml\isoschematron\resources\xsl\RNG2Schtrn.xsl -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl
  copying src\lxml\isoschematron\resources\xsl\XSD2Schtrn.xsl -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl
  creating build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl\iso-schematron-xslt1
  copying src\lxml\isoschematron\resources\xsl\iso-schematron-xslt1\iso_abstract_expand.xsl -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl\iso-schematron-xslt1
  copying src\lxml\isoschematron\resources\xsl\iso-schematron-xslt1\iso_dsdl_include.xsl -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl\iso-schematron-xslt1
  copying src\lxml\isoschematron\resources\xsl\iso-schematron-xslt1\iso_schematron_message.xsl -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl\iso-schematron-xslt1
  copying src\lxml\isoschematron\resources\xsl\iso-schematron-xslt1\iso_schematron_skeleton_for_xslt1.xsl -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl\iso-schematron-xslt1
  copying src\lxml\isoschematron\resources\xsl\iso-schematron-xslt1\iso_svrl_for_xslt1.xsl -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl\iso-schematron-xslt1
  copying src\lxml\isoschematron\resources\xsl\iso-schematron-xslt1\readme.txt -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl\iso-schematron-xslt1
  running build_ext
  building 'lxml.etree' extension
  error: Microsoft Visual C++ 9.0 is required (Unable to find vcvarsall.bat). Get it from http://aka.ms/vcpython27

  ----------------------------------------
←[31m  Failed building wheel for lxml←[0m
Failed to build lxml
Installing collected packages: lxml, webencodings, tinycss2, pre-commit-hooks
  Running setup.py install for lxml
    Complete output from command d:\users\lucas_cimon\.pre-commit\repozp6fda\py_env-default\scripts\python.exe -c "import setuptools, tokenize;__file__='c:\\temp\\pip-build-vjp3v8\\lxml\\setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record c:\temp\pip-mkgjyg-record\install-record.txt --single-version-externally-managed--compile --install-headers d:\users\lucas_cimon\.pre-commit\repozp6fda\py_env-default\include\site\python2.7\lxml:
    Building lxml version 3.5.0.
    Building without Cython.
    ERROR: 'xslt-config' n'est pas reconnu en tant que commande interne
    ou externe, un programme exécutable ou un fichier de commandes.

    ** make sure the development packages of libxml2 and libxslt are installed **

    Using build configuration of libxslt
    running install
    running build
    running build_py
    copying src\lxml\builder.py -> build\lib.win-amd64-2.7\lxml
    copying src\lxml\cssselect.py -> build\lib.win-amd64-2.7\lxml
    copying src\lxml\doctestcompare.py -> build\lib.win-amd64-2.7\lxml
    copying src\lxml\ElementInclude.py -> build\lib.win-amd64-2.7\lxml
    copying src\lxml\pyclasslookup.py -> build\lib.win-amd64-2.7\lxml
    copying src\lxml\sax.py -> build\lib.win-amd64-2.7\lxml
    copying src\lxml\usedoctest.py -> build\lib.win-amd64-2.7\lxml
    copying src\lxml\_elementpath.py -> build\lib.win-amd64-2.7\lxml
    copying src\lxml\__init__.py -> build\lib.win-amd64-2.7\lxml
    copying src\lxml\includes\__init__.py -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\html\builder.py -> build\lib.win-amd64-2.7\lxml\html
    copying src\lxml\html\clean.py -> build\lib.win-amd64-2.7\lxml\html
    copying src\lxml\html\defs.py -> build\lib.win-amd64-2.7\lxml\html
    copying src\lxml\html\diff.py -> build\lib.win-amd64-2.7\lxml\html
    copying src\lxml\html\ElementSoup.py -> build\lib.win-amd64-2.7\lxml\html
    copying src\lxml\html\formfill.py -> build\lib.win-amd64-2.7\lxml\html
    copying src\lxml\html\html5parser.py -> build\lib.win-amd64-2.7\lxml\html
    copying src\lxml\html\soupparser.py -> build\lib.win-amd64-2.7\lxml\html
    copying src\lxml\html\usedoctest.py -> build\lib.win-amd64-2.7\lxml\html
    copying src\lxml\html\_diffcommand.py -> build\lib.win-amd64-2.7\lxml\html
    copying src\lxml\html\_html5builder.py -> build\lib.win-amd64-2.7\lxml\html
    copying src\lxml\html\_setmixin.py -> build\lib.win-amd64-2.7\lxml\html
    copying src\lxml\html\__init__.py -> build\lib.win-amd64-2.7\lxml\html
    copying src\lxml\isoschematron\__init__.py -> build\lib.win-amd64-2.7\lxml\isoschematron
    copying src\lxml\lxml.etree.h -> build\lib.win-amd64-2.7\lxml
    copying src\lxml\lxml.etree_api.h -> build\lib.win-amd64-2.7\lxml
    copying src\lxml\includes\c14n.pxd -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\includes\config.pxd -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\includes\dtdvalid.pxd -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\includes\etreepublic.pxd -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\includes\htmlparser.pxd -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\includes\relaxng.pxd -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\includes\schematron.pxd -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\includes\tree.pxd -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\includes\uri.pxd -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\includes\xinclude.pxd -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\includes\xmlerror.pxd -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\includes\xmlparser.pxd -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\includes\xmlschema.pxd -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\includes\xpath.pxd -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\includes\xslt.pxd -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\includes\etree_defs.h -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\includes\lxml-version.h -> build\lib.win-amd64-2.7\lxml\includes
    copying src\lxml\isoschematron\resources\rng\iso-schematron.rng -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\rng
    copying src\lxml\isoschematron\resources\xsl\RNG2Schtrn.xsl -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl
    copying src\lxml\isoschematron\resources\xsl\XSD2Schtrn.xsl -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl
    copying src\lxml\isoschematron\resources\xsl\iso-schematron-xslt1\iso_abstract_expand.xsl -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl\iso-schematron-xslt1
    copying src\lxml\isoschematron\resources\xsl\iso-schematron-xslt1\iso_dsdl_include.xsl -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl\iso-schematron-xslt1
    copying src\lxml\isoschematron\resources\xsl\iso-schematron-xslt1\iso_schematron_message.xsl -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl\iso-schematron-xslt1
    copying src\lxml\isoschematron\resources\xsl\iso-schematron-xslt1\iso_schematron_skeleton_for_xslt1.xsl -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl\iso-schematron-xslt1
    copying src\lxml\isoschematron\resources\xsl\iso-schematron-xslt1\iso_svrl_for_xslt1.xsl -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl\iso-schematron-xslt1
    copying src\lxml\isoschematron\resources\xsl\iso-schematron-xslt1\readme.txt -> build\lib.win-amd64-2.7\lxml\isoschematron\resources\xsl\iso-schematron-xslt1
    running build_ext
    building 'lxml.etree' extension
    error: Microsoft Visual C++ 9.0 is required (Unable to find vcvarsall.bat).
Get it from http://aka.ms/vcpython27

    ----------------------------------------
←[31mCommand "d:\users\lucas_cimon\.pre-commit\repozp6fda\py_env-default\scripts\python.exe -c "import setuptools, tokenize;__file__='c:\\temp\\pip-build-vjp3v8\\lxml\\setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record c:\temp\pip-mkgjyg-record\install-record.txt --single-version-externally-managed --compile --install-headers d:\users\lucas_cimon\.pre-commit\repozp6fda\py_env-default\include\site\python2.7\lxml" failed with error code 1 in c:\temp\pip-build-vjp3v8\lxml←[0m
```

Leaving apart the fact that the last command fails, could it be possible to handle the encoding of this output under Git Bash (i.e. mysysgit) ?

There are the lines containing non ASCII characters:
```
18:  ou externe, un programme ex[m▒cutable ou un fichier de commandes.
95:←[31m  Failed building wheel for lxml[m▒▒[0m
103:    ou externe, un programme ex[m▒cutable ou un fichier de commandes.
169:←[31mCommand "d:\users\lucas_cimon\.pre-commit\repozp6fda\py_env-default\scripts\python.exe -c "import setuptools, tokenize;__file__='c:\\temp\\pip-build-vjp3v8\\lxml\\setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" install --record c:\temp\pip-mkgjyg-record\install-record.txt --single-version-externally-managed --compile --install-headers d:\users\lucas_cimon\.pre-commit\repozp6fda\py_env-default\include\site\python2.7\lxml" failed with error code 1 in c:\temp\pip-build-vjp3v8\lxml[m▒▒[0m
```

**Comment by asottile** [comment]
Ah dang, I thought this might become a problem at some point.  I'm going to try and make a similar fix to what I did in #248

**Comment by Lucas-C** [comment]
Thanks, I was thinking about that issue too :)

Btw, I switched from Git Bash to MSYS2 and here is how to install `pre-commit ` there, in case you want to document it :

    pacman -S git python gcc libxml2-devel libxslt-devel
    curl --silent --show-error https://bootstrap.pypa.io/get-pip.py | python
    pip install pre-commit

**Comment by asottile** [comment]
Heh, this is more difficult to reproduce under test than the previous error because pip also has the same bug:

```
E           assert 'UnicodeDecodeError' not in '[INFO] Initializ.../pre-commit.log\n'
E             'UnicodeDecodeError' is contained here:
E               [INFO] Initializing environment for /tmp/pytest-of-asottile/pytest-4/test_hook_install_failure0/1.
E               [INFO] Installing environment for /tmp/pytest-of-asottile/pytest-4/test_hook_install_failure0/1.
E               [INFO] Once installed this environment will be reused.
E               [INFO] This may take a few minutes...
E               An unexpected error has occurred: CalledProcessError: Command: ['bash', '-c', ". '/tmp/pytest-of-asottile/pytest-4/test_hook_install_failure0/3/repom5AMjP/py_env-python3.4/bin/activate' && cd '/tmp/pytest-of-asottile/pytest-4/test_hook_install_failure0/3/repom5AMjP/' && pip install ."]
E               Return code: 2
E               Expected return code: 0
E               Output: 
E                   Processing /tmp/pytest-of-asottile/pytest-4/test_hook_install_failure0/3/repom5AMjP
E                   
E               Errors: 
E                   DEPRECATION: --download-cache has been deprecated and will be removed in the future. Pip now automatically uses and configures its cache.
E                   Exception:
E                   Traceback (most recent call last):
E                     File "/tmp/pytest-of-asottile/pytest-4/test_hook_install_failure0/3/repom5AMjP/py_env-python3.4/lib/python3.4/site-packages/pip/compat/__init__.py", line 47, in console_to_str
E                       return s.decode(sys.__stdout__.encoding)
E                   UnicodeDecodeError: 'utf-8' codec can't decode byte 0xb2 in position 3: invalid start byte
E             ?     ++++++++++++++++++
E                   
E                   During handling of the above exception, another exception occurred:
E                   
E                   Traceback (most recent call last):
E                     File "/tmp/pytest-of-asottile/pytest-4/test_hook_install_failure0/3/repom5AMjP/py_env-python3.4/lib/python3.4/site-packages/pip/basecommand.py", line 211, in main
E                       status = self.run(options, args)
E                     File "/tmp/pytest-of-asottile/pytest-4/test_hook_install_failure0/3/repom5AMjP/py_env-python3.4/lib/python3.4/site-packages/pip/commands/install.py", line 305, in run
E                       wb.build(autobuilding=True)
E                     File "/tmp/pytest-of-asottile/pytest-4/test_hook_install_failure0/3/repom5AMjP/py_env-python3.4/lib/python3.4/site-packages/pip/wheel.py", line 705, in build
E                       self.requirement_set.prepare_files(self.finder)
E                     File "/tmp/pytest-of-asottile/pytest-4/test_hook_install_failure0/3/repom5AMjP/py_env-python3.4/lib/python3.4/site-packages/pip/req/req_set.py", line 334, in prepare_files
E                       functools.partial(self._prepare_file, finder))
E                     File "/tmp/pytest-of-asottile/pytest-4/test_hook_install_failure0/3/repom5AMjP/py_env-python3.4/lib/python3.4/site-packages/pip/req/req_set.py", line 321, in _walk_req_to_install
E                       more_reqs = handler(req_to_install)
E                     File "/tmp/pytest-of-asottile/pytest-4/test_hook_install_failure0/3/repom5AMjP/py_env-python3.4/lib/python3.4/site-packages/pip/req/req_set.py", line 505, in _prepare_file
E                       abstract_dist.prep_for_dist()
E                     File "/tmp/pytest-of-asottile/pytest-4/test_hook_install_failure0/3/repom5AMjP/py_env-python3.4/lib/python3.4/site-packages/pip/req/req_set.py", line 123, in prep_for_dist
E                       self.req_to_install.run_egg_info()
E                     File "/tmp/pytest-of-asottile/pytest-4/test_hook_install_failure0/3/repom5AMjP/py_env-python3.4/lib/python3.4/site-packages/pip/req/req_install.py", line 410, in run_egg_info
E                       command_desc='python setup.py egg_info')
E                     File "/tmp/pytest-of-asottile/pytest-4/test_hook_install_failure0/3/repom5AMjP/py_env-python3.4/lib/python3.4/site-packages/pip/utils/__init__.py", line 711, in call_subprocess
E                       line = console_to_str(proc.stdout.readline())
E                     File "/tmp/pytest-of-asottile/pytest-4/test_hook_install_failure0/3/repom5AMjP/py_env-python3.4/lib/python3.4/site-packages/pip/compat/__init__.py", line 49, in console_to_str
E                       return s.decode('utf_8')
E                   UnicodeDecodeError: 'utf-8' codec can't decode byte 0xb2 in position 3: invalid start byte
E                   
E               
E               Check the log at ~/.pre-commit/pre-commit.log
```

**Comment by asottile** [comment]
Ah, I can just use py27 instead of py34 and that reproduces nicely

**Comment by asottile** [comment]
Should be fixed in 0.7.4

**Comment by Lucas-C** [comment]
Thanks !
